generator/generator_tool/generator_tool.cpp

```diff
@@ -245,6 +249,12 @@ int main(int argc, char ** argv)
   if (FLAGS_dump_feature_names != "")
     feature::DumpFeatureNames(datFile, FLAGS_dump_feature_names);
 
+  if (!FLAGS_srtm_path.empty())
```

Comment:
I don't like it.
Consider moving this closer to the FLAGS_generate_search_index check.